pages/jobs/index.js

```diff
@@ -151,6 +160,24 @@ Join us!
               </div>
             )}
           </div>
+          <div className="text-center mt-3 p-4">
+            { !checkOldPost ? (
+              <button className="btn btn-outline btn-lg text-gray-600" onClick={onClickOldPost}>
+                Check closed published jobs
+              </button>
+            ) : (
+            <div className="mt-8 flex flex-col items-stretch sm:rounded-md text-left">
+              <ul className="bg-white shadow overflow-hidden divide-y divide-gray-200 sm:w-2/3 sm:self-center">
+                {closedJobPosts.map((post, index) => (
+                  <JobPostItem key={index} job={post} />
+                ))}
+              </ul>
+              <button className="btn btn-outline-dark back-to-top mt-1 text-gray-600" onClick={onClickOldPost}>
+                Close
```

Comment:
is it the same as `Show closed jobs`?

Reply:
Not really. This just closes the list of jobs and brings the page back to closed state. 

You can check out the netflify preview created above and state if this is necessary.

Reply:
Yeah that sounds a lot more appropriate. Sorry about that !